src/plugins/intel_cpu/tests/functional/subgraph_tests/src/custom_op_zero_shape.cpp

```diff
@@ -0,0 +1,151 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <openvino/op/op.hpp>
+#include <shape_util.hpp>
+#include <shared_test_classes/base/ov_subgraph.hpp>
+#include <ngraph_functions/builders.hpp>
+#include <common_test_utils/ov_tensor_utils.hpp>
+#include "test_utils/cpu_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+
+using namespace ov::test;
+using namespace CPUTestUtils;
+
+namespace CPULayerTestsDefinitions {
+using CustomOpZeroShapeCPUTestParams = std::tuple<ElementType, InputShape>;
+
+class CustomOpZeroShape : public ov::op::Op {
```

Comment:
`CustomOpZeroShape` -> `CustomOpScalar` zero shape may be interpreted also as a so called "zero(empty) tensor".

Reply:
Done